eng/common/pipelines/templates/steps/save-package-properties.yml

```diff
@@ -41,11 +41,26 @@ steps:
           pwsh: true
           workingDirectory: '${{ parameters.WorkingDirectory }}'
 
+      - task: Powershell@2
+        displayName: Save package properties filtered for PR
+        inputs:
+          filePath: ${{ parameters.ScriptDirectory }}/Save-Package-Properties.ps1
+          arguments: >
+            -PrDiff '${{ parameters.DiffDirectory }}/diff.json'
+            -OutDirectory '${{ parameters.PackageInfoDirectory }}'
+          pwsh: true
+          workingDirectory: '${{ parameters.WorkingDirectory }}'
+
       # When running in PR mode, we want the detected changed services to be attached to the build as tags.
       # However, the public identity does not have the permissions to attach tags to the build.
       # Instead, we will save the changed services to a file, attach it as an attachment for PiplineWitness to pick up and utilize.
+      #
       - pwsh: |
-          $changedServices = (Get-Content -Path '${{ parameters.DiffDirectory }}/diff.json' -Raw | ConvertFrom-Json).ChangedServices
```

Comment:
Do you know if the ChangedServices property is still needed in the diff.json file for any other scenarios now?

Reply:
Yeah, `js` does use `ChangedServices`. A couple `Analyze` pieces in other repos use `ChangedServices` as well. I'm torn on just updating the `diff.json` if the two arrays don't align.